Library/Homebrew/extend/pathname.rb

```diff
@@ -343,14 +343,19 @@ def ensure_writable
     chmod saved_perms if saved_perms
   end
 
+  # @private
+  def find_install_info
+    Formula["texinfo"].opt_bin/"install-info" if Formula["texinfo"].any_version_installed?
+  end
+
   # @private
   def install_info
-    quiet_system "/usr/bin/install-info", "--quiet", to_s, "#{dirname}/dir"
+    quiet_system(find_install_info, "--quiet", to_s, "#{dirname}/dir")
```

Comment:
```suggestion
    quiet_system(install_info, "--quiet", to_s, "#{dirname}/dir")
```

Reply:
As @ywwry66 previously pointed out to me, `install_info` is already taken as a method name.

Reply:
@carlocab fixed!